### ISSUE TITLE
eventd-master doesn't build with meson-0.44.0

### ISSUE THREAD

**Issue by Coacher** [sardemff7/eventd#37]
```
meson --buildtype plain --libdir lib64 --localstatedir /var/lib --prefix /usr --sysconfdir /etc --wrap-mode nodownload -Dsystemduserunitdir=/usr/lib/systemd/user -Dsystemdsystemunitdir=/usr/lib/systemd/system -Ddbussessionservicedir=/usr/share/dbus-1/services -Denable-websocket=false -Denable-dns-sd=false -Denable-ssdp=false -Denable-ipv6=true -Denable-systemd=false -Denable-notification-daemon=true -Denable-nd-wayland=false -Denable-nd-xcb=true -Denable-nd-fbdev=false -Denable-im=false -Denable-sound=false -Denable-tts=false -Denable-webhook=false -Denable-libnotify=true -Denable-libcanberra=false -Denable-gobject-introspection=true -Denable-debug=false /var/tmp/portage/net-misc/eventd-0.23.0_p20171112/work/eventd-0.23.0_p20171112 /var/tmp/portage/net-misc/eventd-0.23.0_p20171112/work/eventd-0.23.0_p20171112-build
The Meson build system
Version: 0.44.0
Source dir: /var/tmp/portage/net-misc/eventd-0.23.0_p20171112/work/eventd-0.23.0_p20171112
Build dir: /var/tmp/portage/net-misc/eventd-0.23.0_p20171112/work/eventd-0.23.0_p20171112-build
Build type: native build
Project name: eventd
Native C compiler: x86_64-pc-linux-gnu-gcc (gcc 7.2.0)
Appending CFLAGS from environment: '-O2 -pipe -march=native -Wall'
Appending LDFLAGS from environment: '-Wl,-O1 -Wl,--as-needed'
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/x86_64-pc-linux-gnu-pkg-config (1.3.90)
Native dependency glib-2.0 found: YES 2.52.3
Native dependency gobject-2.0 found: YES 2.52.3
Native dependency gio-2.0 found: YES 2.52.3
Native dependency gio-unix-2.0 found: YES 2.52.3
Native dependency gmodule-2.0 found: YES 2.52.3
Has header "locale.h": YES
Has header "fcntl.h": YES
Has header "errno.h": YES
Has header "sys/socket.h": YES
Configuring config.h using configuration
Configuring config.ent using configuration
Compiler for C supports argument -Wformat=2: YES
Compiler for C supports argument -Wno-unused-parameter: YES

Executing subproject libnkutils.

Project name: libnkutils
Native C compiler: x86_64-pc-linux-gnu-gcc (gcc 7.2.0)
Native dependency uuid found: YES 2.31.0
Native dependency xkbcommon found: YES 0.7.2
Program git found: YES (/usr/bin/git)
Build targets in project: 14

Subproject libnkutils finished.
Program g-ir-scanner found: YES (/usr/bin/g-ir-scanner)
Program g-ir-compiler found: YES (/usr/bin/g-ir-compiler)
Native dependency gobject-introspection-1.0 found: YES 1.52.1
Has header "sys/un.h": YES
Has header "netinet/in.h": YES
Native dependency libsoup-2.4 found: YES 2.58.2
Dependency avahi-client found: NO
Dependency avahi-glib found: NO
Native dependency gssdp-1.0 found: YES 1.0.2
Configuring org.eventd.fdo-notifications.service using configuration
Native dependency gdk-pixbuf-2.0 found: YES 2.36.10
Native dependency cairo found: YES 1.14.12
Native dependency pango found: YES 1.40.12
Native dependency pangocairo found: YES 1.40.12
Library m found: YES

Meson encountered an error in file plugins/nd/meson.build, line 79, column 4:
Subproject name must not contain a path separator.
```

**Comment by sardemff7** [comment]
Known regression of mesonbuild/meson#2794, waiting for Meson upstream to make a decision about that.

**Comment by Coacher** [comment]
Fixed in meson-0.44.1.
